temporalio/bridge/src/runtime.rs

```diff
@@ -27,16 +29,8 @@ pub(crate) struct Runtime {
 
 #[derive(FromPyObject)]
 pub struct TelemetryConfig {
-    tracing: Option<TracingConfig>,
```

Comment:
Removed as part of https://github.com/temporalio/sdk-core/pull/599 which didn't work anyways and, based on internal discussions, was a rarely used advanced core debug tracer that we wanted to remove instead of deprecate.